.openzeppelin/unknown-80002.json

```diff
@@ -39,6 +39,16 @@
       "address": "0x19875eA86503734f2f9Ed461463e0312A3b42563",
       "txHash": "0xba5002593532e73a88ef5ed5f12013abebb196c10a455d66d604e96199471475",
       "kind": "transparent"
+    },
+    {
+      "address": "0x4ae40104C1a1117C4b533c06b0be7884747eFa72",
+      "txHash": "0x884039d09d0aee1217f6225b66cefa950eabfefbf19570ee5f9ddf127ba6f300",
+      "kind": "transparent"
+    },
+    {
+      "address": "0xB7314d56A505554aB54ADab6bF56b6b82d182EB9",
+      "txHash": "0xbff77c246856f675ae6e97e204d68e83ebb6c5423d7c63bf9782eb8ed994fef1",
+      "kind": "transparent"
```

Comment:
I see something is deployed. We don't keep records that we didn't deploy ourselves in this file.

Reply:
Change reverted